framework/include/meshgenerators/SidesetsEncloseBlocks.h

```diff
@@ -0,0 +1,30 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "MeshGenerator.h"
+
+/**
+ * MeshGenerator that checks if a set of blocks is enclosed by a set of sidesets.
+ * It can either add sides that are not covered by a sideset by a new sidesets or
```

Comment:
Seems like some word is missing. Did you mean something like: "...that are not covered by a sideset by _specifying_ a new sideset..."?
If so, then `addClassDescription` in `InputParamters` needs to be updated accordingly, as well.

Reply:
The descriptions are actually identical.